src/components/EnrolledLearnersForInactiveCoursesTable/data/tests/constants.js

```diff
@@ -0,0 +1,38 @@
+const mockUseCourseEnrollments = {
```

Comment:
I already created these mock enrollments at src/components/LearnerActivityTable/data/tests/constants.js. As these are common so I think we can move it to some parent common place. Currently, the same code is being duplicated.